ibmnode/4/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# (C) Copyright IBM Corporation 2016, 2017
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM ubuntu:16.04
+
+MAINTAINER Bethany Griggs <bethany.griggs@uk.ibm.com> (@BethGriggs)
+
+ENV IBM_VERSION 4.8.4.0
+ENV NODE_VERSION 4.8.4
+
+RUN buildDeps='wget ca-certificates unzip openjdk-8-jre'; \
+    apt-get update \
+      && apt-get install -y --no-install-recommends $buildDeps \
```

Comment:
Minor change - Please fix the spaces to align with the rest of the RUN command.

Reply:
Indenting follow-on lines is pretty common no? Helps make it clear that it's not part of the same line.